Documentation/Blazorise.Docs/Pages/Docs/Extensions/DataGrid/GettingStartedPage.razor

```diff
@@ -255,8 +255,8 @@
     <DocsAttributesItem Name="SortMode" Type="DataGridSortMode" Default="Multiple">
         Defines whether the user can sort only by one column or by multiple.
     </DocsAttributesItem>
-    <DocsAttributesItem Name="SortOrderChanged" Type="EventCallback<DataGridSortOrderChangedEventArgs>" Default="null">
-        Occurs after the column sorting has changed.
+    <DocsAttributesItem Name="SortDefinitionChanged" Type="EventCallback<DataGridSortDefinitionChangedEventArgs>" Default="null">
```

Comment:
@stsrki do you also like "**SortDefintion**" prefix? I'm kinda on the fence about it. But i can't think of any other... naming is hard... :d Maybe **SortInfo**? Or you think this is fine?

Reply:
Or maybe event `Sorting`Changed?

Reply:
If the sort, in general, has changed, then `Sorting`Changed is a good name.

Reply:
I'll change this.

Reply:
Fixed in acef337e31e4427a9bf736801eb5f16c9624a59d